main.go

```diff
@@ -161,6 +162,29 @@ func main() {
 	dg.Close()
 }
 
+func logCommand(s *discordgo.Session, gID, user, command string) {
+	// Add slash prefix to the log if given command is a slash command
+	// Else add prefix as BotPrefix
+	var prefix = "Slash"
+	if strings.Contains(command, botPrefix) {
+		prefix = "BotPrefix"
+	}
+
+	// Log commands in the channel
+	if logChannelID != "" {
+		guild, err := s.Guild(gID)
+		if err != nil {
+			fmt.Printf("Error getting guild info: %s", err)
+			return
+		}
+
+		s.ChannelMessageSend(
+			logChannelID,
+			fmt.Sprintf(`[%s] %s > %s > %s`, prefix, guild.Name, user, command),
```

Comment:
This looks great, making a reusable function was a good idea. A small change here would be nice though.

Currently, the bot prints the following:
`[Slash] Warfstache's server > Wilford Warfstache > time`

I'd like it to be a _little_ more specific by printing this instead:
`[Slash] Warfstache's server > Wilford Warfstache#0256 > time`

That way, we can make sure to narrow a log down to the exact user who used the command.

Reply:
Sure, will change it.